### DISCUSSION TITLE
This Code Search and Navigation is really bad

### DISCUSSION THREAD

**Discussion by furry72** [community/community#54278] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

How to disable the new Code Search and Navigation gui

**Comment by zautumnz** [comment, marked as answer]
On desktop, click your account icon, click Feature Preview near the bottom, go to Code Search, click Disable.

**Reply by furry72** [reply]
But, now it's impossible disable, when Feature Preview, isn't appear Code Search

**Reply by schnappi-wkl** [reply]
i also encountered this promble，the new code searching navigateion is so bad

**Reply by furry72** [reply]
I hate the new Code Search, when search something of GitHub, appear weird projects

**Reply by am0089** [reply]
any other solution to getting back the old code search with sort last indexed ?

**Reply by furry72** [reply]
are a monster Github, has change the code search and view for a angry users

**Reply by furry72** [reply]
Please github change this horrible code search

**Reply by JohnyP36** [reply]
@github Why you just removed this option. I really hate the new layout and I can't revert it 🤮😤🤬

**Reply by zautumnz** [reply]
@JohnyP36 I don't work at Github, not much I can do about it. Maybe you could try [emailing the CEO directly](https://github.com/ashtom). I've given up on Github having good UX.

**Comment by MikeRich88** [comment]
No more reverting. You just have to deal with it now.

**"We don't care. We don't have to. We're the ~~Phone~~ Code Company."**

**Comment by DmitryMigunov** [comment]
![image](https://github.com/community/community/assets/1397194/dd0cc0a5-fffc-45ad-b40f-f6fa8f1a51b6)
Nice :) Firefox 118.0.2

**Reply by AdamShwert** [reply]
Are you using any plugins that modify github in some way? Something that changes tab width or something like that?

**Reply by DmitryMigunov** [reply]
No, I deactivated all plugins, but nothing changed.

**Reply by AdamShwert** [reply]
What OS are you on?

**Reply by DmitryMigunov** [reply]
arch 6.5.7, i3

**Reply by AdamShwert** [reply]
I've done some investigating and unfortunately haven't been able to isolate the cause, but in the meantime, if you turn on `wrap lines` in the `...` menu at the top right of a code file page, this should no longer be an issue.

**Comment by JohnyP36** [comment]
@github @ashtom Why did you just removed this option in the feature preview. I really hate the new layout and I can't revert it 🤮😤🤬

**Comment by aznashwan** [comment]
+1, on latest Safari on macOS it takes a good 3 seconds or more for a link with a commit+line number (e.g. [this random sample from something I'm working on](https://github.com/microsoft/windows-container-networking/blob/d502b1b6d7e9e59f744dbc4903c3fa986668e7a4/common/core/network.go#L404)) to open properly.

The page twitches, flashes, and writhes as it loads the side-bar, main code view, and hopefully after 3-5 SECONDS it will actually highlight  the line(s) of code you actually want to see.

Please please PLEASE give us an option to simply disable the code indexing etc.

I imagine most people can already navigate code way better through their editors/IDEs, and the common usecase of opening a link to a piece of code should not have to suffer so a small handful who prefer navigating directly in the site get to click on symbols and never leave their browser...

**Reply by aznashwan** [reply]
Screencap for reference here:

https://github.com/community/community/assets/6192535/215f879f-b403-426f-bb53-2f4787294e3c
